### ISSUE TITLE
The SQL Dump file doesn't work in my testing

### ISSUE THREAD

**Issue by jackkinsella** [edavis10/heroku_s3_backup#1]
Hi there,

This looks like a great script for automated backups. I've tried it out and it does indeed backup daily. However when I do disaster recovery testing by trying to open the dump files in psql, they give syntax errors: 

<code>
ERROR:  syntax error at or near "PGDMP" 
LINE 1: PGDMP         ^ 
GRANT 
GRANT 
ERROR:  syntax error at or near "id" 
LINE 1: id integer NOT NULL,         ^
 ERROR:  syntax error at or near "" 
LINE 1:     START WITH 1         ^ E
RROR:  syntax error at or near ""
LINE 1:     id integer NOT NULL,         

etc. etc.
</code>

I was wondering if this is a problem with how the script dumps or just one with my schema. Perhaps it could be incompatibility?Have you had similar problems?

Thanks
Jack

**Comment by trevorturk** [comment]
Please re-read the blog post more closely around the section and restoring from the backup. It mentions how to use pg_restore, which is necessary. Then it should work for you. 

edavis10, you can close this ticket, I think.

**Comment by jackkinsella** [comment]
Hi Trevor,

Sorry I scanned your post and didn't see the bit about using pg_restore. I should have known better and read things more carefully! 

I tried the pg_restore option as you mentioned on your blog but now I am getting another error:

<pre><code>
pg_restore: [archiver (db)] Error while PROCESSING TOC:
pg_restore: [archiver (db)] Error from TOC entry 1540; 1259 353486 TABLE acts_as_xapian_jobs iofrvzsmpa
pg_restore: [archiver (db)] could not execute query: ERROR:  role "iofrvzsmpa" does not exist
    Command was: ALTER TABLE public.acts_as_xapian_jobs OWNER TO iofrvzsmpa;
pg_restore: [archiver (db)] Error from TOC entry 1539; 1259 353484 SEQUENCE acts_as_xapian_jobs_id_seq iofrvzsmpa
pg_restore: [archiver (db)] could not execute query: ERROR:  role "iofrvzsmpa" does not exist
    Command was: ALTER TABLE public.acts_as_xapian_jobs_id_seq OWNER TO iofrvzsmpa;
pg_restore: [archiver (db)] Error from TOC entry 1550; 1259 474983 TABLE babysitters 
.... etc.
</code></pre>


Any ideas?

**Comment by trevorturk** [comment]
Yeah, I mention something about it in the blog post just under the part about pg_restore ;)
